asv_bench/benchmarks/series_methods.py

```diff
@@ -201,4 +201,75 @@ def time_series_datetimeindex_repr(self):
         getattr(self.s, 'a', None)
 
 
+class All(object):
+
+    params = [[10**3, 10**6], ['fast', 'slow']]
+    param_names = ['N', 'case']
+
+    def setup(self, N, case):
+        val = case != 'fast'
+        self.s = Series([val] * N)
+
+    def time_all(self, N, case):
+        self.s.all()
+
+
+class Any(object):
+
+    params = [[10**3, 10**6], ['fast', 'slow']]
+    param_names = ['N', 'case']
+
+    def setup(self, N, case):
+        val = case == 'fast'
+        self.s = Series([val] * N)
+
+    def time_any(self, N, case):
+        self.s.any()
+
+
+class NanOps(object):
+
+    params = [[10**3, 10**6], ['int64', 'float64']]
+    param_names = ['N', 'dtype']
+
+    def setup(self, N, dtype):
+        self.s = Series([1] * N, dtype=dtype)
+
+    def time_var(self, N, dtype):
```

Comment:
could make the function another param

Reply:
I'll probably remove these prior to merge as they duplicate existing benchmarks.

Reply:
can you update this